modules/price/src/CommerceCurrencyListBuilder.php

```diff
@@ -2,10 +2,10 @@
 
 /**
  * @file
- * Contains \Drupal\commerce_price\Entity\Controller\CommerceCurrencyListBuilder.
+ * Contains \Drupal\commerce_price\Entity\CommerceCurrencyListBuilder.
```

Comment:
the "Entity\" part of this namespace is incorrect, let's fix that as well. So it should be: Contains \Drupal\commerce_price\CommerceCurrencyListBuilder.